web_development_101/the_front_end/project_html_css.md

```diff
@@ -95,6 +95,7 @@ Submit a pull request with a link to your solution in this [file](https://github
   <summary> Show Student Solutions </summary>
 
 * Add your solution below this line!
+* https://idavidson-95.github.io/google-homepage/
```

Comment:
Your solution should be formatted with markdown similar to the thousands of other solutions below where you inserted your link.

Once that's done, we can display your solution. If you have questions, do not hesitate to ask here or in  our Discord community